### ISSUE TITLE
The page url's return not found 

### ISSUE THREAD

**Issue by kumarvermaankit** [kareem-m/photography-portfolio#1]
When I try to navigate to about , index , gallery , it says page not found .

![image](https://user-images.githubusercontent.com/56042513/226987499-da9e8a2d-13da-43a4-b7d8-72044e5a4fe3.png)

**Comment by kareem-m** [comment]
First thanks for this issues, When you use the code and start a local server you should write in the end of link ".html" .

This because there is a file named ".htaccess" in the "public" folder, this make the link shorter by cancelling the ".html" from the link and make it better and shorter but this work only in a wide range server not local server.

And finally i'm working to solve this problem.

**Comment by kumarvermaankit** [comment]
Yeah I had already found this error and fixed it several times , but
somehow it changes back to original code again . Can you help with this?
One more thing , if you wish , I would like contribute.

**Comment by kareem-m** [comment]
I will try solving this problem but not now.

Could you please send a link for your facebook profile, i want to message
you in messenger to improve my English and my programming skills.

Or you can message me here https://www.facebook.com/kareem1911

**Comment by kumarvermaankit** [comment]
Hey Actually , I don't use fb a lot , are you on instagram or linkedin?

**Comment by kareem-m** [comment]
Ok on Instagram.

This is my profile :
https://instagram.com/kareemmoh1911?igshid=ZDdkNTZiNTM=

**Comment by kumarvermaankit** [comment]
Hey , I've followed you on instagram

**Comment by lakz-ph** [comment]
hii i think here some html files are missing!
